mlflow/store/artifact/models_artifact_repo.py

```diff
@@ -80,12 +80,15 @@ def split_models_uri(uri):
         """
         Split 'models:/<name>/<version>/path/to/model' into
         ('models:/<name>/<version>', 'path/to/model').
+        Split 'models:/<name>@alias/path/to/model' into
+        ('models:/<name>@alias', 'path/to/model').
         """
         path = urllib.parse.urlparse(uri).path
-        if path.count("/") >= 3 and not path.endswith("/"):
+        if path.count("/") >= 2 and not path.endswith("/"):
```

Comment:
just for my own understanding, why do we have the condition that the path can't end with `/`?

Reply:
Emm maybe it can, there's no difference though. But a well-formatted model_uri should be `models:/<model_name>/model_version`.